README.md

```diff
@@ -84,6 +84,15 @@ Binary will be created as `<path to your test file.nim>.bin` under the `build` d
 make test/tests/common/test_enr_builder.nim
 ```
 
+### Formatting Code
+
+We use [`nph`](https://github.com/arnetheduck/nph) to format nim code.
+You can install a convenient pre-commit git hook with the following command:
+
+```
+make install_nph_prehook
+```
```

Comment:
Considering that the hook will always be triggered, we don't need the developer to explicitly do `make install_nph_prehook`. In case my other comment makes sense of course :)
```suggestion
```

Reply:
I don't think we should install pre-hooks nilly-willy, especially if a dev has already their own prehooks in place. Which is why I set it up as separate step.